src/kernel/arch/aarch64/tty.zig

```diff
@@ -83,12 +84,14 @@ fn writeString(str: []const u8) !void {
     }
 }
 
-fn setCursor(x: u8, y: u8) void {
+pub fn clearScreen() void {}
```

Comment:
If this function is intended to do nothing then I suggest removing it and setting the `.clear` field in the returned `TTY` instance to `null` to instead use the default screen clearing function.

Reply:
Done